daemon/daemon_unix.go

```diff
@@ -248,12 +248,11 @@ func (daemon *Daemon) adaptContainerSettings(hostConfig *containertypes.HostConf
 		hostConfig.ShmSize = container.DefaultSHMSize
 	}
 	var err error
-	if hostConfig.SecurityOpt == nil {
-		hostConfig.SecurityOpt, err = daemon.generateSecurityOpt(hostConfig.IpcMode, hostConfig.PidMode, hostConfig.Privileged)
-		if err != nil {
-			return err
-		}
+	opts, err := daemon.generateSecurityOpt(hostConfig.IpcMode, hostConfig.PidMode, hostConfig.Privileged)
```

Comment:
The pause container has:
```
11:27:42 [src/k8s.io/kubernetes] ‹master› docker inspect 8dcc | grep -i label
        "MountLabel": "system_u:object_r:container_file_t:s0:c301,c564",
        "ProcessLabel": "system_u:system_r:container_t:s0:c301,c564",
            "Labels": {
```
and etcd container has:
```
11:23:59 [src/k8s.io/kubernetes] ‹master› docker inspect 1b23 | grep -i label
        "MountLabel": "system_u:object_r:container_file_t:s0:c231,c542",
        "ProcessLabel": "system_u:system_r:container_t:s0:c231,c542",
                "label:type:spc_t",
                "label=user:system_u",
                "label=role:system_r",
                "label=type:container_t",
                "label=level:s0:c301,c564"
            "Labels": {
```
Both containers are created with:
```
\"SecurityOpt\":[\"seccomp=unconfined\",\"label:type:spc_t\"]
```
The etcd container join the ipc ns of the pause container. Afterwards it gets process labels from the pause container (apparently container_t) and appends them to the etcd container labels